### DISCUSSION TITLE
Symfony Deployer.php

### DISCUSSION THREAD

**Discussion by MrMariusWang** [symfony/symfony#49909] [Q&A]
Hey guys,
I'm currently trying to deploy to a web server for my first time. I use Deployer to handle this. 
Now I have the problem that my deploy.php is failing all the time.
Can somebody give me an example how to build the deploy.php? (My deploy:update_code fails : [develop]   The "release_path" ... does not exist. Even if the release_path exists)

Thanks for your help! :)

**Comment by ghost** [comment]
ask on deployer discussions https://github.com/deployphp/deployer/discussions
